src/m365/flow/commands/run/run-list.ts

```diff
@@ -71,6 +126,20 @@ class FlowRunListCommand extends AzmgmtItemsListCommand<{ name: string, startTim
       this.handleRejectedODataJsonPromise(err);
     }
   }
+
+  private getFilters(status?: string, triggerStartTime?: string, triggerEndTime?: string): string[] {
```

Comment:
According to me, will be cleaner if we just pass the whole CommandOptions object and determine in this function which options we need. It will be more future proof if we need more filters.